python/taichi/aot/record.py

```diff
@@ -18,10 +18,39 @@ def record_action_config(key, value):
 
 
 def start_recording(filename):
+    """Start recording kernel information to a `yml` file.
```

Comment:
Here and below: All the verbs should be in third person.

```suggestion
    """Starts recording kernel information to a `yml` file.
```